src/app/core/auth/auth-request.service.ts

```diff
@@ -27,14 +27,21 @@ export abstract class AuthRequestService {
               ) {
   }
 
-  protected fetchRequest(request: RestRequest): Observable<RemoteData<AuthStatus>> {
-    return this.rdbService.buildFromRequestUUID<AuthStatus>(request.uuid).pipe(
+  protected fetchRequest(request: RestRequest, ...linksToFollow: FollowLinkConfig<AuthStatus>[]): Observable<RemoteData<AuthStatus>> {
+    return this.rdbService.buildFromRequestUUID<AuthStatus>(request.uuid, ...linksToFollow).pipe(
       getFirstCompletedRemoteData(),
     );
   }
 
-  protected getEndpointByMethod(endpoint: string, method: string): string {
-    return isNotEmpty(method) ? `${endpoint}/${method}` : `${endpoint}`;
+  protected getEndpointByMethod(endpoint: string, method: string, ...linksToFollow: FollowLinkConfig<AuthStatus>[]): string {
+    let url = isNotEmpty(method) ? `${endpoint}/${method}` : `${endpoint}`;
+    if (linksToFollow && linksToFollow.length > 0) {
```

Comment:
@atarix83 : LGTM is saying the first part of this `if` is a "useless conditional" because `linksToFollow` will always be `true` at this point.

Reply:
@atarix83 : See the rule description from LGTM which it links tohttps://lgtm.com/rules/7860090/

In the text of that it notes..
> Note that in JavaScript (unlike some other languages) arrays and objects are always considered to be true when evaluated in a Boolean context. The code should instead check `.length`:

Essentially, since `linksToFollow` is an array, I think it always will return `true`.

Reply:
fixed